qiskit_experiments/library/randomized_benchmarking/__init__.py

```diff
@@ -64,4 +64,5 @@
 from .rb_utils import RBUtils
 from .clifford_synthesis import RBDefaultCliffordSynthesis
 from .layer_fidelity import LayerFidelity
+from .layer_fidelity_unitary import LayerFidelityUnitary
```

Comment:
This needs to appear in the docstring above alongside `LayerFidelity` in order to appear in the docs. Also, it needs to be referenced in `qiskit_experiments/library/__init__.py` to appear on the page listing all the experiments.

Also, most experiments get lifted up into the `qiskit_experiments.library` namespace, though I see `LayerFidelity` is not one of them. So currently you have to do `from qiskit_experiments.library.randomized_benchmarking import LayerFidelity` instead of `from qiskit_experiments.library import LayerFidelity`. I don't love how `import qiskit_experiments` ends up importing everything but that is the way it has been set up so far.

Reply:
Ok, I think this is fixed

Reply:
You still might want to add `LayerFidelityUnitary` to the docstring above on the line below `LayerFidelity`. That would make it show up on [this page about RB experiments](https://qiskit-community.github.io/qiskit-experiments/apidocs/mod_randomized_benchmarking.html).

By the way, would `UnitaryLayerFidelity` make more sense as the name?

Reply:
I think done now?